### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by lotharschulz**
contribution details added

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+little bit helps, and credit will always be given.
+
+You can contribute in many ways:
+
+## Types of Contributions
+
+### Report Bugs
+
+
+Report bugs at [GitHub Issues](https://github.com/zalando-stups/lizzy/issues)
```
Comment (rashamalek) [inline CONTRIBUTING.md]: https://github.com/zalando/ghe-backup/issues

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+
+If you are reporting a bug, please include:
+
+* Any details about your setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+### Fix Bugs
+
+
+Look through the [GitHub Issues](https://github.com/zalando-stups/lizzy/issues) for bugs. Anything tagged with [bug](https://github.com/zalando/ghe-backup/issues?utf8=%E2%9C%93&q=label%3Abug%20)
```
Comment (rashamalek) [inline CONTRIBUTING.md]: the same

**Comment by lotharschulz** [comment]
👍

**Comment by rashamalek** [comment]
👍

**Pull request merged**